### PR TITLE
Make mvtnorm dependency instead of suggested

### PR DESCRIPTION
**Pull request by erikcs**
This is a solid self-contained package that is no hassle having as a dependency - and we get easy access to construct test DGPs in docstings/examples/test cases.

~`mvtnorm` is used to generate test data, it is easy to do this with [Cholesky](https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Drawing_values_from_the_distribution) so here is just a 
suggested added utility function instead.~

~(that way we can have tests that use this DGP included on CRAN without requiring `mvtnorm` )~

### CODE REVIEW

**Comment by davidahirshberg** [comment]
You can just call stuff like this rmvnorm. It's not exported, so there's no need to follow any kind of namespacing convention.

**Comment by erikcs** [comment]
Ok, you want to use `library(mvtnorm)` in the README/vignette, and the internal mvnorm in tests?

**Comment by davidahirshberg** [comment]
I guess, even though it's a bit weird.  I didn't really think the vignette thing through before making that comment.  This is a pretty bad vignette, so it probably makes sense to plan for it to go away.

**Comment by erikcs** [comment]
Makes sense, let's make a self contained vignette/example later that we also can have in the docstring

**Comment by erikcs** [comment]
This is fine, no?  We can hold onto that dgp in the existing tests without worrying about mvtnorm + CRAN

**Comment by swager** [comment]
Sorry to chime in the middle. My take on this is that `mvtnorm` is a super-solid package; almost just base R. See https://cran.r-project.org/web/packages/mvtnorm/index.html. I have no problem having an import for mvtnorm in synthdid; and introducing our our implementations of functions for random Gaussians may end up being a bigger cost in terms of code quality.

**Comment by erikcs** [comment]
Yes I agree and thought about the same, the only issue I saw was that mvtnorm requires compilation and synthdid does not - and I thought you wanted to keep it that way.

But if that is fine I can just make `mvtnorm` a dependency instead

**Comment by swager** [comment]
Ohh interesting -- I hadn't realized that was a trade-off.

Still, though, my guess is that -- in the long run -- we'll need to introduce some compiled code into synthdid regardless, so this doesn't seem to be too big a cost.

On the other hand, `rmvnorm` seems like one of these superficially-simple functions that actually require quite a bit of work to get just right (how does it handle very large matrices, very small or large values, etc.), and the `mvtnorm` version will persumably have been tested against all kinds of things we've never even thought of. So there's upside to using it.

**Comment by erikcs** [comment]
Sounds good! Let's make `mvnorm` a dependency! (a light one it is)

File: DESCRIPTION [hunk 1]
```diff
@@ -11,10 +11,10 @@ Description: Estimate average treatment effects in panel data.
     Currently provides methods only for the case that all treated units
     adopt treatment at the same time.
 Depends:
-    R (>= 3.4)
+    R (>= 3.4),
+    mvtnorm
```
Comment (swager) [inline DESCRIPTION:15]: I believe standard practice here would be to use "imports" as opposed to "depends", so we don't glob all the namespaces together. Then we can call the function as `mvtnorm::rmvnorm`.

https://stackoverflow.com/questions/8637993/better-explanation-of-when-to-use-imports-depends

**Review by swager — APPROVED** [review]
looks good to me!

**Pull request merged**